hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupCommands.java

```diff
@@ -142,6 +142,12 @@ public void execute() throws IOException {
         throw new IOException(INCORRECT_USAGE);
       }
 
+      if (cmdline.hasOption(OPTION_YARN_QUEUE_NAME)) {
+        String queueName = cmdline.getOptionValue(OPTION_YARN_QUEUE_NAME);
+        // Set MR job queuename to configuration
+        getConf().set("mapreduce.job.queuename", queueName);
```

Comment:
Also isn't this change need for BackupDriver? I do see it is being parsed there.

Reply:
Currently, all the parameter parsing for the backup command is done in BackupCommands.java. If we separately move the parsing of the yarn queue to BackupDriver.java, the code might not be as elegant.

Reply:
Got it, thanks for the clarification!